packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -253,10 +253,10 @@ export const ChatScreen = () => {
   useEffect(() => {
     dispatch(fetchBlockees())
     dispatch(fetchBlockers())
-    if (otherUser.user_id) {
+    if (otherUser?.user_id) {
       dispatch(fetchPermissions({ userIds: [otherUser.user_id] }))
     }
-  }, [chatId, dispatch, otherUser.user_id])
+  }, [chatId, dispatch, otherUser])
```

Comment:
I think we want to keep this as `otherUser.user_id` if possible so it only changes when that prop changes?